package.json

```diff
@@ -29,13 +29,16 @@
     "flow-bin": "^0.33.0",
     "lodash": "^4.15.0",
     "nyc": "^8.1.0",
+    "rollup": "^0.36.3",
+    "rollup-plugin-babel": "^2.6.1",
+    "rollup-plugin-node-resolve": "^2.0.0",
```

Comment:
This is required to be able to import an `index` file without having to explicitly list `./foo/index`.